src/docfx/config/Config.cs

```diff
@@ -150,7 +151,12 @@ private static Config LoadCore(string configPath, CommandLineOptions options = n
         private static JObject LoadOriginalConfigObject(string configPath, List<string> parents, bool expand)
         {
             // TODO: support URL
-            var (_, config) = YamlUtility.Deserialize<JObject>(File.ReadAllText(configPath));
+            var (errors, config) = YamlUtility.Deserialize<JObject>(File.ReadAllText(configPath));
+            if (errors.Any())
```

Comment:
>errors.Any() [](start = 16, length = 12)

will it be null?

Reply:
nope, will  return an empty list if no error